tests/test_jobs.py

```diff
@@ -56,10 +54,9 @@ def test_restrictions():
     jobs.error('a', {'key': 'a2'}, 'error')
     jobs.error('b', {'key': 'b1'}, 'error')
 
-    assert_true(len(jobs & 'table_name = "a"') == 2, 'There should be two entries for table a')
-    assert_true(len(jobs & 'status = "error"') == 2, 'There should be two entries with error status')
-    assert_true(len(jobs & 'table_name = "a"' & 'status = "error"') == 1,
-                'There should be only one entries with error status in table a')
+    assert_true(len(jobs & {'table_name': "a"}) == 2)
```

Comment:
why did you get rid of the error message here?

Reply:
because it carried no useful information. The message is not useful without looking at the code. If do look at the code, then the assert condition is just as readable as the error message.

Reply:
Yeah fair